marketdata/charts.go

```diff
@@ -28,6 +32,9 @@ func InitCharts() *Charts {
 
 func (c *Charts) GetChartData(coin uint, token string, currency string, timeStart int64) (blockatlas.ChartData, error) {
 	chartsData := blockatlas.ChartData{}
+	if timeStart < minUnixData {
+		timeStart = minUnixData
```

Comment:
`max(timeStart, minUnixData)` like this

Reply:
Do you mean to create a method to verify the max value?

```
func max(timeStart, min int64) int64 {
	if timeStart < min {
		return min
	}
	return timeStart
}
```

Reply:
There is a function for `Max()`: https://github.com/trustwallet/blockatlas/blob/master/pkg/numbers/number.go#L10